### PR TITLE
docs: create readme docs for collaborators

### PR DESCRIPTION
**Pull request by HJunyuan**
**What does this PR do?**

- ~~Collate all collaborator related documentation into a centralised location~~ Documentation now in [repo Wiki](/Notarise-gov-sg/api-notarise-healthcerts/wiki) to deprecate old documentation here: https://github.com/Open-Attestation/schemata/pull/38
- Update main `README.md`
- Update ART sample to include modality
- Add `fhirBundle` diagram image and source file

### CODE REVIEW

File: docs/4-recommended-coding-set.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <th>Display</th>
+    <th>EU DCC</th>
+    <th>GPay COVID Card</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="2">"PCR"</td>
+    <td>94531-1</td>
+    <td>SARS-CoV-2 (COVID-19) RNA panel - Respiratory specimen by NAA with probe detection</td>
```
Comment (zawmyolatt) [inline docs/4-recommended-coding-set.md:21]: it is possible to change display name as `SARS-CoV-2 (COVID-19) RNA [Presence] in Specimen by NAA with probe detection` also ? Current EU schema not allow test name `nm` not more than 80 chars.

Comment (HJunyuan) [reply docs/4-recommended-coding-set.md:21]: The display field shown here is for following the LOINC long common name: https://loinc.org/94531-1/

I thought [EU schema has its own display naming](https://github.com/ehn-dcc-development/ehn-dcc-schema/blob/release/1.3.1/valuesets/test-type.json)?

```json
{
  "LP6464-4": {
    "display": "Nucleic acid amplification with probe detection",
    "lang": "en",
    "active": true,
    "version": "2.69",
    "system": "http://loinc.org"
  },
  "LP217198-3": {
    "display": "Rapid immunoassay",
    "lang": "en",
    "active": true,
    "version": "2.69",
    "system": "http://loinc.org"
  }
}
```

In any case, we don't validate the display field at this moment. Collaborators can always specify any display field of any length. Suggest that the EU DCC generation just trim if > 80 characters?

Comment (zawmyolatt) [reply docs/4-recommended-coding-set.md:21]: yea.. can trim in eu dcc generator , but don't know where should cut off name like last part or something. And for PCR, eu dcc accept extra test name string along with PCR test type code  `LP6464-4`

**Comment by HJunyuan** [comment]
I'll be moving the documentation over to the [repo Wiki](/Notarise-gov-sg/api-notarise-healthcerts/wiki) instead.

**Review by zawmyolatt — APPROVED** [review]

**Pull request merged**